{"name": "test_integrity_procedure_manufacturer_insert", "status": "broken", "statusDetails": {"message": "_mysql_connector.MySQLInterfaceError: Commands out of sync; you can't run this command now", "trace": "db_connection = <mysql.connector.connection_cext.CMySQLConnection object at 0x106980730>\n\n    def test_integrity_procedure_manufacturer_insert(db_connection):\n        with allure.step('Проверка обеспечения целостности данных при добавлении данных через процедуру'):\n            with db_connection as connection:\n                cursor = connection.cursor()\n                cursor.execute(\"call Manufacturer_insert(%s, %s)\", ('Fake', 4))\n                assert cursor.fetchone() is not None\n>               connection.commit()\n\ntest_db.py:67: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <mysql.connector.connection_cext.CMySQLConnection object at 0x106980730>\n\n    def commit(self) -> None:\n        \"\"\"Commit current transaction\"\"\"\n        if self._cmysql:\n            self.handle_unread_result()\n>           self._cmysql.commit()\nE           _mysql_connector.MySQLInterfaceError: Commands out of sync; you can't run this command now\n\n../../../../Library/Python/3.9/lib/python/site-packages/mysql/connector/connection_cext.py:517: MySQLInterfaceError"}, "steps": [{"name": "Проверка обеспечения целостности данных при добавлении данных через процедуру", "status": "broken", "statusDetails": {"message": "_mysql_connector.MySQLInterfaceError: Commands out of sync; you can't run this command now\n", "trace": "  File \"/Users/akopotov/Documents/College/Testing/checkpoint_6_dB_test/test_db.py\", line 67, in test_integrity_procedure_manufacturer_insert\n    connection.commit()\n  File \"/Users/akopotov/Library/Python/3.9/lib/python/site-packages/mysql/connector/connection_cext.py\", line 517, in commit\n    self._cmysql.commit()\n"}, "start": 1698417481927, "stop": 1698417481928}], "start": 1698417481927, "stop": 1698417481930, "uuid": "b5dac806-708c-48b8-9980-541dd78e8df9", "historyId": "aea949440cc6df0b3b90fab2029dc84b", "testCaseId": "aea949440cc6df0b3b90fab2029dc84b", "fullName": "test_db#test_integrity_procedure_manufacturer_insert", "labels": [{"name": "suite", "value": "test_db"}, {"name": "host", "value": "alexMBP-14.local"}, {"name": "thread", "value": "45446-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_db"}]}